scipy/stats/_pagel.py

```diff
@@ -0,0 +1,287 @@
+
+    .. math::
+
+        \chi_L^2 = \frac{\left[12L-3mn(n+1)^2\right]^2}{mn^2(n^2-1)(n+1)}
+
+    "which is distributed approximately as chi-square with 1 degree of
+    freedom. The ordinary use of :math:`\chi^2` tables would be
+    equivalent to a two-sided test of agreement. If a one-sided test
+    is desired, *as will almost always be the case*, the probability
+    discovered in the chi-square table should be *halved*."
+
+    However, this standardized statistic does not distinguish between the
+    observed values being well correlated with the predicted ranks and being
+    _anti_-correlated with the predicted ranks. Instead, we follow [2]_
+    and calculate the standardized statistic
+
+    .. math::
+
+        \Lambda = \frac{L - E_0}{\sqrt{V_0}},
+
+    where :math:`E_0 = \frac{1}{4} mn(n+1)^2` and
+    :math:`V_0 = \frac{1}{144} mn^2(n+1)(n^2-1)`, "which is asymptotically
+    normal under the null hypothesis". The asymptotic calculation is not
+    adjusted for the possibility of ties, which, according to [2]_, does not
+    violate the significance level of the result.
+
+    The *p*-value for ``method='exact'`` is generated by comparing the observed
+    value of :math:`L` against the :math:`L` values generated for all
+    :math:`(n!)^m` possible permutations of ranks. For ``method='mc'``,
+    the observed value of :math:`L`  is compared against ``n_s``
+    randomly sampled rank permutations. If the calculation is to be adjusted
+    for the possibility of ties, then the observed :math:`L` is compared
+    against the :math:`L` values generated for all  :math:`(m^m)^n` possible
+    tables of ranks (`exact`) or a randomly generated subset of them (`mc`).
+
+
+    References
+    ----------
+    .. [1] Ellis Batten Page, "Ordered hypotheses for multiple treatments:
+       a significant test for linear ranks", *Journal of the American
+       Statistical Association* 58(301), p. 216--230, 1963.
+
+    .. [2] Markus Neuhauser, *Nonparametric Statistical Test: A computational
+       approach*, CRC Press, p. 150--152, 2012.
+
+    .. [3] Statext LLC, "Page's L Trend Test - Easy Statistics", *Statext -
+       Statistics Study*, https://www.statext.com/practice/PageTrendTest03.php,
+       Accessed July 12, 2020.
+
+    Examples
+    --------
+    We use the example from [3]_: 10 students are asked to rate three
+    teaching methods - tutorial, lecture, and seminar - on a scale of 1-5,
+    with 1 being the lowest and 5 being the highest. We have decided that
+    a confidence level of 99% is required to reject the null hypothsis in favor
+    of our alternative: that the seminar will have the highest ratings and the
+    tutorial will have the lowest. Initially, the data have been tabulated with
+    each row representing an individual student's ratings of the three methods
+    in the following order: tutorial, lecture, seminar.
+
+    >>> table = [[3, 4, 3],
+                 [2, 2, 4],
```

Comment:
minor: this requires `...` at the start of each line to be valid doctest syntax.

Reply:
Yup, thanks!